src/it/java/com/zotov/edu/passportofficerestservice/model/PassportRequest.java

```diff
@@ -0,0 +1,18 @@
+package com.zotov.edu.passportofficerestservice.model;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.With;
+
+@Data
+@Builder
+public class PassportRequest {
```

Comment:
Let it be PassportPostRequest since we have PassportPutRequest :)

Reply:
Renamed